{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "executionInfo": {
     "elapsed": 7710,
     "status": "ok",
     "timestamp": 1637389375650,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "SdrKCQ5mdrYn",
    "outputId": "b15522a9-4073-4001-9acc-cd135cc724cf"
   },
   "outputs": [],
   "source": [
    "# !pip install bioc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "executionInfo": {
     "elapsed": 69708,
     "status": "ok",
     "timestamp": 1637389083710,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "d4jy_S20cbD3",
    "outputId": "4c2b0df8-cda7-4435-9e24-39bb076e1580"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mounted at /content/drive\n",
      "/content/drive\n",
      "/content/drive/My Drive\n",
      "/content/drive/My Drive/Colab Notebooks\n",
      "/content/drive/My Drive/Colab Notebooks/Dr.Kojima_JOB\n"
     ]
    }
   ],
   "source": [
    "# from google.colab import drive\n",
    "# drive.mount('/content/drive')\n",
    "# %cd drive\n",
    "# %cd \"My Drive\"\n",
    "# %cd \"Colab Notebooks\"\n",
    "# # /content/drive/My Drive/Colab Notebooks\n",
    "# %cd \"Dr.Kojima_JOB\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "id": "jifdZZbcTK80"
   },
   "source": [
    "# BioCCollectionオブジェクト調査"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "id": "2TdCYC2seR13"
   },
   "outputs": [],
   "source": [
    "import bioc\n",
    "\n",
    "# bioc.bioc.BioCCollection型\n",
    "with open(\"data_original/aimed_bioc.xml\", 'r') as fp:\n",
    "    anem_bioc = bioc.load(fp)\n",
    "\n",
    "# # bioc.biocxml.decoder.BioCXMLDocumentReader型\n",
    "# reader = bioc.BioCXMLDocumentReader(\"data/anem_bioc.xml\")\n",
    "\n",
    "# # bioc.bioc.BioCDocument型\n",
    "# for document in reader:\n",
    "#   print(document)\n",
    "#   print(type(document))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "executionInfo": {
     "elapsed": 648,
     "status": "ok",
     "timestamp": 1636957686160,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "JRHrSQ_sRuSX",
    "outputId": "a63b1edb-a1a9-4747-fcb5-a9d444ff576d"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "infons : <class 'dict'>\n",
      "encoding : <class 'str'>\n",
      "version : <class 'str'>\n",
      "standalone : <class 'bool'>\n",
      "source : <class 'str'>\n",
      "date : <class 'str'>\n",
      "key : <class 'str'>\n",
      "documents : <class 'list'>\n"
     ]
    }
   ],
   "source": [
    "for key, value in anem_bioc.__dict__.items():\n",
    "    print(key, ':',type(value))\n",
    "#     print(key, ':', value,type(value))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 199
    },
    "executionInfo": {
     "elapsed": 650,
     "status": "ok",
     "timestamp": 1636957101543,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "AWUTWX8CPtA8",
    "outputId": "806a6da3-aeff-4e99-9231-af99b0671376"
   },
   "outputs": [],
   "source": [
    "# anem_bioc.__str__()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "id": "qZ5G5A53SPG2"
   },
   "outputs": [],
   "source": [
    "document = anem_bioc.__dict__[\"documents\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "executionInfo": {
     "elapsed": 310,
     "status": "ok",
     "timestamp": 1637286130118,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "WlNn-ruwXgoJ",
    "outputId": "349c0745-a590-4257-c1fe-a0f0cb001923"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "infons : <class 'dict'>\n",
      "annotations : <class 'list'>\n",
      "relations : <class 'list'>\n",
      "id : <class 'str'>\n",
      "passages : <class 'list'>\n"
     ]
    }
   ],
   "source": [
    "for key, value in document[0].__dict__.items():\n",
    "    print(key, ':',type(value))\n",
    "#     print(key, ':', value,type(value))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "executionInfo": {
     "elapsed": 328,
     "status": "ok",
     "timestamp": 1636959376885,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "fY2MF3duYTfn",
    "outputId": "d70231ab-dc58-4fd6-cbb4-8c8e33e5fe05"
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "bioc.bioc.BioCPassage"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "type(document[0].__dict__[\"passages\"][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "executionInfo": {
     "elapsed": 315,
     "status": "ok",
     "timestamp": 1637391162576,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "SwEh601scrlZ"
   },
   "outputs": [],
   "source": [
    "a_BioCPassage_list = document[0].__dict__[\"passages\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "executionInfo": {
     "elapsed": 13,
     "status": "ok",
     "timestamp": 1636960523257,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "PzhWpFECYebK",
    "outputId": "a4f696a7-02e9-45bf-8201-36cc6f0b891d"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "infons : {} <class 'dict'>\n",
      "annotations : [BioCAnnotation[id=T1,text='delta-catenin',infons=[file=ann,type=protein],locations=[BioCLocation[offset=19,length=13]],], BioCAnnotation[id=T3,text='presenilin (PS) 1',infons=[file=ann,type=protein],locations=[BioCLocation[offset=122,length=17]],], BioCAnnotation[id=T2,text='presenilin 1',infons=[file=ann,type=protein],locations=[BioCLocation[offset=66,length=12]],], BioCAnnotation[id=T6,text='PS1',infons=[file=ann,type=protein],locations=[BioCLocation[offset=415,length=3]],], BioCAnnotation[id=T7,text='PS-2',infons=[file=ann,type=protein],locations=[BioCLocation[offset=441,length=4]],], BioCAnnotation[id=T4,text='delta-catenin',infons=[file=ann,type=protein],locations=[BioCLocation[offset=182,length=13]],], BioCAnnotation[id=T5,text='delta-catenin',infons=[file=ann,type=protein],locations=[BioCLocation[offset=313,length=13]],], BioCAnnotation[id=T10,text='PS1',infons=[file=ann,type=protein],locations=[BioCLocation[offset=513,length=3]],], BioCAnnotation[id=T11,text='armadillo-protein',infons=[file=ann,type=protein],locations=[BioCLocation[offset=618,length=17]],], BioCAnnotation[id=T12,text='p0071',infons=[file=ann,type=protein],locations=[BioCLocation[offset=637,length=5]],], BioCAnnotation[id=T8,text='PS1',infons=[file=ann,type=protein],locations=[BioCLocation[offset=474,length=3]],], BioCAnnotation[id=T13,text='PS1',infons=[file=ann,type=protein],locations=[BioCLocation[offset=660,length=3]],], BioCAnnotation[id=T9,text='PS2',infons=[file=ann,type=protein],locations=[BioCLocation[offset=482,length=3]],]] <class 'list'>\n",
      "relations : [BioCRelation[id=R4,infons=[relation type=Interaction,file=ann,type=Relation],nodes=[BioCNode[refid=T12,role=Arg1],BioCNode[refid=T13,role=Arg2]],], BioCRelation[id=R2,infons=[relation type=Interaction,file=ann,type=Relation],nodes=[BioCNode[refid=T3,role=Arg1],BioCNode[refid=T4,role=Arg2]],], BioCRelation[id=R3,infons=[relation type=Interaction,file=ann,type=Relation],nodes=[BioCNode[refid=T5,role=Arg1],BioCNode[refid=T6,role=Arg2]],], BioCRelation[id=R1,infons=[relation type=Interaction,file=ann,type=Relation],nodes=[BioCNode[refid=T1,role=Arg1],BioCNode[refid=T2,role=Arg2]],]] <class 'list'>\n",
      "offset : 0 <class 'int'>\n",
      "text : Isolation of human delta-catenin and its binding specificity with presenilin 1.\n",
      "We screened proteins for interaction with presenilin (PS) 1, and cloned the full-length cDNA of human delta-catenin, which encoded 1225 amino acids.\n",
      "Yeast two-hybrid assay, GST binding assay and immunoprecipitation demonstrated that delta-catenin interacted with a hydrophilic loop region in the endoproteolytic C-terminal fragment of PS1, but not with that of PS-2.\n",
      "These results suggest that PS1 and PS2 partly differ in function.\n",
      "PS1 loop fragment containing the pathogenic mutation retained the binding ability.\n",
      "We also found another armadillo-protein, p0071, interacted with PS1. <class 'str'>\n",
      "sentences : [] <class 'list'>\n"
     ]
    }
   ],
   "source": [
    "for key, value in a_BioCPassage_list[0].__dict__.items():\n",
    "  print(key, ':', value,type(value))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {
    "executionInfo": {
     "elapsed": 314,
     "status": "ok",
     "timestamp": 1637391209389,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "UB6sbH33c2WU"
   },
   "outputs": [],
   "source": [
    "a_BioCAnnotation_list = a_BioCPassage_list[0].__dict__[\"annotations\"]\n",
    "a_BioCRelations_list = a_BioCPassage_list[0].__dict__[\"relations\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "executionInfo": {
     "elapsed": 362,
     "status": "ok",
     "timestamp": 1637391232164,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "OQWhPnKBHjoC",
    "outputId": "acd1d75c-59c8-4219-845f-0d8b4dc7433f"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "infons : {'file': 'ann', 'type': 'protein'} <class 'dict'>\n",
      "locations : [BioCLocation[offset=19,length=13]] <class 'list'>\n",
      "id : T1 <class 'str'>\n",
      "text : delta-catenin <class 'str'>\n"
     ]
    }
   ],
   "source": [
    "for key, value in a_BioCAnnotation_list[0].__dict__.items():\n",
    "  print(key, ':', value,type(value))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "executionInfo": {
     "elapsed": 9,
     "status": "ok",
     "timestamp": 1636960770026,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "RWQodaZTdt75",
    "outputId": "219c30f6-33d7-4319-93e9-ee27c0ac042d"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "infons : {'relation type': 'Interaction', 'file': 'ann', 'type': 'Relation'} <class 'dict'>\n",
      "id : R4 <class 'str'>\n",
      "nodes : [BioCNode[refid=T12,role=Arg1], BioCNode[refid=T13,role=Arg2]] <class 'list'>\n"
     ]
    }
   ],
   "source": [
    "for key, value in a_BioCRelations_list[0].__dict__.items():\n",
    "  print(key, ':', value,type(value))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "a_BioCRelations_Nodes = a_BioCRelations_list[0].__dict__[\"nodes\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "Nodes_set = set()\n",
    "for BioCNode in a_BioCRelations_Nodes:\n",
    "    print(BioCNode.__dict__[\"refid\"])\n",
    "    Nodes_set.add(BioCNode.__dict__[\"refid\"])\n",
    "    \n",
    "Nodes_set"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "id": "jifdZZbcTK80"
   },
   "source": [
    "# データ作成"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import bioc\n",
    "import itertools\n",
    "import csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "id": "iA_57GoDgtck"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "aimed_bioc.xml\n",
      "bioinfer_bioc.xml\n",
      "hprd50_bioc.xml\n",
      "iepa_bioc.xml\n",
      "lll_bioc.xml\n"
     ]
    }
   ],
   "source": [
    "files = os.listdir(\"data_original\")\n",
    "\n",
    "val_name = []\n",
    "for a_file in files:\n",
    "  print(a_file)\n",
    "  with open(\"data_original/\"+a_file, 'r') as fp:\n",
    "    dynmc_name = a_file.replace(\".xml\",\"\")\n",
    "    locals()[dynmc_name] = bioc.load(fp)\n",
    "    val_name.append(dynmc_name)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "executionInfo": {
     "elapsed": 667,
     "status": "ok",
     "timestamp": 1637395039114,
     "user": {
      "displayName": "Y trmt",
      "photoUrl": "https://lh3.googleusercontent.com/a-/AOh14GgDQvHXLMLM4izDdj4TwI9Yh1z_2PwieV4cnzJESA=s64",
      "userId": "18315248992010109137"
     },
     "user_tz": -540
    },
    "id": "bO9D1qXlA7XW"
   },
   "outputs": [],
   "source": [
    "def get_PairData(filename):\n",
    "    # bioc.biocxml.decoder.BioCXMLDocumentReader型\n",
    "    all_data = []\n",
    "    reader = bioc.BioCXMLDocumentReader(\"data_original/\"+filename+\".xml\")\n",
    "\n",
    "    # bioc.bioc.BioCDocument型\n",
    "    for document in reader:\n",
    "        document_id = document.__dict__[\"id\"]\n",
    "\n",
    "        for passage in document.__dict__[\"passages\"]:\n",
    "            text = passage.__dict__[\"text\"]\n",
    "\n",
    "            ann_ids_texts = []\n",
    "            for annotation in passage.__dict__[\"annotations\"]:\n",
    "                ann_id = annotation.__dict__[\"id\"]\n",
    "                ann_text = annotation.__dict__[\"text\"]\n",
    "                ann_ids_texts.append([ann_id,ann_text])\n",
    "\n",
    "            for pair in itertools.combinations(ann_ids_texts, 2):\n",
    "                a,b = list(pair)\n",
    "\n",
    "                row = [document_id,text,a[0],b[0],a[1],b[1],set([a[0],b[0]])]\n",
    "                all_data.append(row)\n",
    "                \n",
    "    return all_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_RelDataDict(filename):\n",
    "    # bioc.biocxml.decoder.BioCXMLDocumentReader型\n",
    "    all_rel_data_dict = dict()\n",
    "    reader = bioc.BioCXMLDocumentReader(\"data_original/\"+filename+\".xml\")\n",
    "    \n",
    "    for document in reader:\n",
    "        document_id = document.__dict__[\"id\"]\n",
    "        document_rels = []\n",
    "\n",
    "        for passage in document.__dict__[\"passages\"]:\n",
    "\n",
    "            for relations in passage.__dict__[\"relations\"]:\n",
    "                a_node = set()\n",
    "                for nodes in relations.__dict__[\"nodes\"]:\n",
    "                    a_node.add(nodes.__dict__[\"refid\"])\n",
    "                document_rels.append(a_node)\n",
    "\n",
    "        all_rel_data_dict[document_id] = document_rels\n",
    "    return all_rel_data_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "# filename = \"aimed_bioc.xml\"\n",
    "\n",
    "# PairData = get_PairData(filename)\n",
    "# RelDataDict = get_RelDataDict(filename)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "id": "WK80wRhkYTsC"
   },
   "outputs": [],
   "source": [
    "def Create_CSV(filename):\n",
    "    PairData = get_PairData(filename)\n",
    "    RelDataDict = get_RelDataDict(filename)\n",
    "    \n",
    "    newPairData = []\n",
    "    for pair in PairData:\n",
    "        TorF = \"FALSE\"\n",
    "        if pair[-1] in RelDataDict[pair[0]]:\n",
    "            TorF = \"TRUE\"\n",
    "        newPairData.append(pair[:-1]+[TorF])\n",
    "        \n",
    "    with open(\"data/\"+filename+\".csv\", \"w\") as f:\n",
    "        writer = csv.writer(f, lineterminator='\\n')\n",
    "        writer.writerows(newPairData)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "Create_CSV(\"lll_bioc\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "aimed_bioc\n",
      "bioinfer_bioc\n",
      "hprd50_bioc\n",
      "iepa_bioc\n",
      "lll_bioc\n"
     ]
    }
   ],
   "source": [
    "for filename in val_name:\n",
    "    print(filename)\n",
    "    Create_CSV(filename)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "ename": "SyntaxError",
     "evalue": "invalid syntax (<ipython-input-9-b13ef1f1b4ca>, line 3)",
     "output_type": "error",
     "traceback": [
      "\u001b[1;36m  File \u001b[1;32m\"<ipython-input-9-b13ef1f1b4ca>\"\u001b[1;36m, line \u001b[1;32m3\u001b[0m\n\u001b[1;33m    l = [row if row != []: else: pass for row in reader]\u001b[0m\n\u001b[1;37m                         ^\u001b[0m\n\u001b[1;31mSyntaxError\u001b[0m\u001b[1;31m:\u001b[0m invalid syntax\n"
     ]
    }
   ],
   "source": [
    "with open('data/lll_bioc.csv') as f:\n",
    "    reader = csv.reader(f)\n",
    "    l = [row for row in reader if row != []]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[['LLL_d7',\n",
       "  'Expression of the sigma(K)-dependent cwlH gene depended on gerE.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'sigma(K)',\n",
       "  'cwlH',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d7',\n",
       "  'Expression of the sigma(K)-dependent cwlH gene depended on gerE.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigma(K)',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d7',\n",
       "  'Expression of the sigma(K)-dependent cwlH gene depended on gerE.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'cwlH',\n",
       "  'gerE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d0',\n",
       "  'In vivo studies of the activity of four of the kinases, KinA, KinC, KinD (ykvD) and KinE (ykrQ), using abrB transcription as an indicator of Spo0A~P level, revealed that KinC and KinD were responsible for Spo0A~P production during the exponential phase of growth in the absence of KinA and KinB.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'KinC',\n",
       "  'Spo0A~P',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d0',\n",
       "  'In vivo studies of the activity of four of the kinases, KinA, KinC, KinD (ykvD) and KinE (ykrQ), using abrB transcription as an indicator of Spo0A~P level, revealed that KinC and KinD were responsible for Spo0A~P production during the exponential phase of growth in the absence of KinA and KinB.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'KinC',\n",
       "  'KinD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d0',\n",
       "  'In vivo studies of the activity of four of the kinases, KinA, KinC, KinD (ykvD) and KinE (ykrQ), using abrB transcription as an indicator of Spo0A~P level, revealed that KinC and KinD were responsible for Spo0A~P production during the exponential phase of growth in the absence of KinA and KinB.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'Spo0A~P',\n",
       "  'KinD',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T6',\n",
       "  'T7',\n",
       "  'sigmaF',\n",
       "  'SpoIIAA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T6',\n",
       "  'T4',\n",
       "  'sigmaF',\n",
       "  'SpoIIAA',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T6',\n",
       "  'T5',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T6',\n",
       "  'T1',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T6',\n",
       "  'T3',\n",
       "  'sigmaF',\n",
       "  'SpoIIAB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T6',\n",
       "  'T2',\n",
       "  'sigmaF',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T7',\n",
       "  'T4',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIAA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T7',\n",
       "  'T5',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T7',\n",
       "  'T1',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T7',\n",
       "  'T3',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIAB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T7',\n",
       "  'T2',\n",
       "  'SpoIIAA',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T4',\n",
       "  'T5',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIAB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'SpoIIAA',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T5',\n",
       "  'T1',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T5',\n",
       "  'T3',\n",
       "  'SpoIIE',\n",
       "  'SpoIIAB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T5',\n",
       "  'T2',\n",
       "  'SpoIIE',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'SpoIIE',\n",
       "  'SpoIIAB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'SpoIIE',\n",
       "  'sigmaF',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d9',\n",
       "  'Taken together, the results support the idea that sequestration of SpoIIE protein into the prespore plays an important role in the control of sigmaF activation and in coupling this activation to septation.\\nsigmaF is controlled by a regulatory cascade involving an anti-sigma factor, SpoIIAB, an anti-anti-sigma factor, SpoIIAA, and a membrane-bound phosphatase, SpoIIE, which converts the inactive, phosphorylated form of SpoIIAA back to the active form.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'SpoIIAB',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d40',\n",
       "  'Although spo0H is transcriptionally regulated by the key regulatory device that controls sporulation initiation, the Spo0 phosphorelay, there is considerable evidence implicating a mechanism of post-translational control that governs the activity and concentration of sigmaH.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'Spo0',\n",
       "  'spo0H',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T6',\n",
       "  'T7',\n",
       "  'sigmaA',\n",
       "  'clpC',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T6',\n",
       "  'T4',\n",
       "  'sigmaA',\n",
       "  'CtsR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T6',\n",
       "  'T5',\n",
       "  'sigmaA',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T6',\n",
       "  'T1',\n",
       "  'sigmaA',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T6',\n",
       "  'T3',\n",
       "  'sigmaA',\n",
       "  'clpC',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T6',\n",
       "  'T2',\n",
       "  'sigmaA',\n",
       "  'sigmaA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T7',\n",
       "  'T4',\n",
       "  'clpC',\n",
       "  'CtsR',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T7',\n",
       "  'T5',\n",
       "  'clpC',\n",
       "  'sigmaB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T7',\n",
       "  'T1',\n",
       "  'clpC',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T7',\n",
       "  'T3',\n",
       "  'clpC',\n",
       "  'clpC',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T7',\n",
       "  'T2',\n",
       "  'clpC',\n",
       "  'sigmaA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T4',\n",
       "  'T5',\n",
       "  'CtsR',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'CtsR',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'CtsR',\n",
       "  'clpC',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'CtsR',\n",
       "  'sigmaA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T5',\n",
       "  'T1',\n",
       "  'sigmaB',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T5',\n",
       "  'T3',\n",
       "  'sigmaB',\n",
       "  'clpC',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T5',\n",
       "  'T2',\n",
       "  'sigmaB',\n",
       "  'sigmaA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'sigmaB',\n",
       "  'clpC',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigmaB',\n",
       "  'sigmaA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d35',\n",
       "  'The Bacillus subtilis clpC operon is regulated by two stress induction pathways relying on either sigmaB or a class III stress induction mechanism acting at a sigmaA-like promoter.\\nOur data demonstrate that the CtsR protein acts as a global repressor of the clpC operon, as well as other class III heat shock genes, by preventing unstressed transcription from either the sigmaB- or sigmaA-dependent promoter and might be inactivated or dissociate under inducing stress conditions.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'clpC',\n",
       "  'sigmaA',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d32',\n",
       "  'Transcriptional studies demonstrated that the expression from the araE promoter is induced by L-arabinose, repressed by glucose, and negatively regulated by AraR.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'AraR',\n",
       "  'araE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d23',\n",
       "  'Transcriptional studies showed that nadE is strongly induced in response to heat, ethanol and salt stress or after starvation for glucose in a sigma B-dependent manner.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigma B',\n",
       "  'nadE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d24',\n",
       "  'Furthermore, YtxH and YvyD seemed to be induced after phosphate starvation in the wild type in a sigma(B)-dependent manner and in the sigB mutant probably via sigma(H).',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'YvyD',\n",
       "  'sigma(B)',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d24',\n",
       "  'Furthermore, YtxH and YvyD seemed to be induced after phosphate starvation in the wild type in a sigma(B)-dependent manner and in the sigB mutant probably via sigma(H).',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'YvyD',\n",
       "  'YtxH',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d24',\n",
       "  'Furthermore, YtxH and YvyD seemed to be induced after phosphate starvation in the wild type in a sigma(B)-dependent manner and in the sigB mutant probably via sigma(H).',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'YvyD',\n",
       "  'sigma(H)',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d24',\n",
       "  'Furthermore, YtxH and YvyD seemed to be induced after phosphate starvation in the wild type in a sigma(B)-dependent manner and in the sigB mutant probably via sigma(H).',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'sigma(B)',\n",
       "  'YtxH',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d24',\n",
       "  'Furthermore, YtxH and YvyD seemed to be induced after phosphate starvation in the wild type in a sigma(B)-dependent manner and in the sigB mutant probably via sigma(H).',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigma(B)',\n",
       "  'sigma(H)',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d24',\n",
       "  'Furthermore, YtxH and YvyD seemed to be induced after phosphate starvation in the wild type in a sigma(B)-dependent manner and in the sigB mutant probably via sigma(H).',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'YtxH',\n",
       "  'sigma(H)',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'sigmaK',\n",
       "  'gerE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigmaK',\n",
       "  'cot',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T6',\n",
       "  'sigmaK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T7',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T4',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T5',\n",
       "  'sigmaK',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T10',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T11',\n",
       "  'sigmaK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T12',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T8',\n",
       "  'sigmaK',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T13',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T14',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T9',\n",
       "  'sigmaK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T15',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T16',\n",
       "  'sigmaK',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T17',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T18',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T19',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T1',\n",
       "  'T20',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'gerE',\n",
       "  'cot',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T6',\n",
       "  'gerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T7',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T4',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T5',\n",
       "  'gerE',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T10',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T11',\n",
       "  'gerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T12',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T8',\n",
       "  'gerE',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T13',\n",
       "  'gerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T14',\n",
       "  'gerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T9',\n",
       "  'gerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T15',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T16',\n",
       "  'gerE',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T17',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T18',\n",
       "  'gerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T19',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T3',\n",
       "  'T20',\n",
       "  'gerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T6',\n",
       "  'cot',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T7',\n",
       "  'cot',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T4',\n",
       "  'cot',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T5',\n",
       "  'cot',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T10',\n",
       "  'cot',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T11',\n",
       "  'cot',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T12',\n",
       "  'cot',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T8',\n",
       "  'cot',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T13',\n",
       "  'cot',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T14',\n",
       "  'cot',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T9',\n",
       "  'cot',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T15',\n",
       "  'cot',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T16',\n",
       "  'cot',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T17',\n",
       "  'cot',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T18',\n",
       "  'cot',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T19',\n",
       "  'cot',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T2',\n",
       "  'T20',\n",
       "  'cot',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T7',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T4',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T5',\n",
       "  'sigmaK',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T10',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T11',\n",
       "  'sigmaK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T12',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T8',\n",
       "  'sigmaK',\n",
       "  'gerE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T13',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T14',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T9',\n",
       "  'sigmaK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T15',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T16',\n",
       "  'sigmaK',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T17',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T18',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T19',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T6',\n",
       "  'T20',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T4',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T5',\n",
       "  'GerE',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T10',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T11',\n",
       "  'GerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T12',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T8',\n",
       "  'GerE',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T13',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T14',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T9',\n",
       "  'GerE',\n",
       "  'sigmaK',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T15',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T16',\n",
       "  'GerE',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T17',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T18',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T19',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T7',\n",
       "  'T20',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T5',\n",
       "  'GerE',\n",
       "  'sigK',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T10',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T11',\n",
       "  'GerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T12',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T8',\n",
       "  'GerE',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T13',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T14',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T9',\n",
       "  'GerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T15',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T16',\n",
       "  'GerE',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T17',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T18',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T19',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T4',\n",
       "  'T20',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T10',\n",
       "  'sigK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T11',\n",
       "  'sigK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T12',\n",
       "  'sigK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T8',\n",
       "  'sigK',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T13',\n",
       "  'sigK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T14',\n",
       "  'sigK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T9',\n",
       "  'sigK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T15',\n",
       "  'sigK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T16',\n",
       "  'sigK',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T17',\n",
       "  'sigK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T18',\n",
       "  'sigK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T19',\n",
       "  'sigK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T5',\n",
       "  'T20',\n",
       "  'sigK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T11',\n",
       "  'GerE',\n",
       "  'sigmaK',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T12',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T8',\n",
       "  'GerE',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T13',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T14',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T9',\n",
       "  'GerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T15',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T16',\n",
       "  'GerE',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T17',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T18',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T19',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T10',\n",
       "  'T20',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T12',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T8',\n",
       "  'sigmaK',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T13',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T14',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T9',\n",
       "  'sigmaK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T15',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T16',\n",
       "  'sigmaK',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T17',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T18',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T19',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T11',\n",
       "  'T20',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T8',\n",
       "  'GerE',\n",
       "  'gerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T13',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T14',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T9',\n",
       "  'GerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T15',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T16',\n",
       "  'GerE',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T17',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T18',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T19',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T12',\n",
       "  'T20',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T8',\n",
       "  'T13',\n",
       "  'gerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T8',\n",
       "  'T14',\n",
       "  'gerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T8',\n",
       "  'T9',\n",
       "  'gerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T8',\n",
       "  'T15',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T8',\n",
       "  'T16',\n",
       "  'gerE',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T8',\n",
       "  'T17',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T8',\n",
       "  'T18',\n",
       "  'gerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T8',\n",
       "  'T19',\n",
       "  'gerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T8',\n",
       "  'T20',\n",
       "  'gerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T13',\n",
       "  'T14',\n",
       "  'cotD',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T13',\n",
       "  'T9',\n",
       "  'cotD',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T13',\n",
       "  'T15',\n",
       "  'cotD',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T13',\n",
       "  'T16',\n",
       "  'cotD',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T13',\n",
       "  'T17',\n",
       "  'cotD',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T13',\n",
       "  'T18',\n",
       "  'cotD',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T13',\n",
       "  'T19',\n",
       "  'cotD',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T13',\n",
       "  'T20',\n",
       "  'cotD',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T14',\n",
       "  'T9',\n",
       "  'cotD',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T14',\n",
       "  'T15',\n",
       "  'cotD',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T14',\n",
       "  'T16',\n",
       "  'cotD',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T14',\n",
       "  'T17',\n",
       "  'cotD',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T14',\n",
       "  'T18',\n",
       "  'cotD',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T14',\n",
       "  'T19',\n",
       "  'cotD',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T14',\n",
       "  'T20',\n",
       "  'cotD',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T9',\n",
       "  'T15',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T9',\n",
       "  'T16',\n",
       "  'sigmaK',\n",
       "  'sigK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T9',\n",
       "  'T17',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T9',\n",
       "  'T18',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T9',\n",
       "  'T19',\n",
       "  'sigmaK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T9',\n",
       "  'T20',\n",
       "  'sigmaK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T15',\n",
       "  'T16',\n",
       "  'GerE',\n",
       "  'sigK',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T15',\n",
       "  'T17',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T15',\n",
       "  'T18',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T15',\n",
       "  'T19',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T15',\n",
       "  'T20',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T16',\n",
       "  'T17',\n",
       "  'sigK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T16',\n",
       "  'T18',\n",
       "  'sigK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T16',\n",
       "  'T19',\n",
       "  'sigK',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T16',\n",
       "  'T20',\n",
       "  'sigK',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T17',\n",
       "  'T18',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T17',\n",
       "  'T19',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T17',\n",
       "  'T20',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T18',\n",
       "  'T19',\n",
       "  'cotD',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T18',\n",
       "  'T20',\n",
       "  'cotD',\n",
       "  'cotD',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d18',\n",
       "  'Most cot genes, and the gerE gene, are transcribed by sigmaK RNA polymerase.\\nPreviously, it was shown that the GerE protein inhibits transcription in vitro of the sigK gene encoding sigmaK.\\nThese results demonstrate that sigmaK-dependent transcription of gerE initiates a negative feedback loop in which GerE acts as a repressor to limit production of sigmaK.\\nA low level of GerE activated transcription of cotD by sigmaK RNA polymerase in vitro, but a higher level of GerE repressed cotD transcription.\\nHere, we show that GerE binds near the sigK transcriptional start site, to act as a repressor.\\nWe show that GerE binds to two sites that span the -35 region of the cotD promoter.\\nThese results suggest that a rising level of GerE in sporulating cells may first activate cotD transcription from the upstream site then repress transcription as the downstream site becomes occupied.',\n",
       "  'T19',\n",
       "  'T20',\n",
       "  'GerE',\n",
       "  'cotD',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d11',\n",
       "  'In addition to controlling transcription of phrC, sigmaH appears to control expression of at least one other gene required for production of CSF.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigmaH',\n",
       "  'phrC',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T6',\n",
       "  'T7',\n",
       "  'ydaE',\n",
       "  'ydaG',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T6',\n",
       "  'T4',\n",
       "  'ydaE',\n",
       "  'ydaP',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T6',\n",
       "  'T5',\n",
       "  'ydaE',\n",
       "  'sigmaB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T6',\n",
       "  'T8',\n",
       "  'ydaE',\n",
       "  'yfkM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T6',\n",
       "  'T1',\n",
       "  'ydaE',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T6',\n",
       "  'T3',\n",
       "  'ydaE',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T6',\n",
       "  'T2',\n",
       "  'ydaE',\n",
       "  'bmrUR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T7',\n",
       "  'T4',\n",
       "  'ydaG',\n",
       "  'ydaP',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T7',\n",
       "  'T5',\n",
       "  'ydaG',\n",
       "  'sigmaB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T7',\n",
       "  'T8',\n",
       "  'ydaG',\n",
       "  'yfkM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T7',\n",
       "  'T1',\n",
       "  'ydaG',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T7',\n",
       "  'T3',\n",
       "  'ydaG',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T7',\n",
       "  'T2',\n",
       "  'ydaG',\n",
       "  'bmrUR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T4',\n",
       "  'T5',\n",
       "  'ydaP',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T4',\n",
       "  'T8',\n",
       "  'ydaP',\n",
       "  'yfkM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'ydaP',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'ydaP',\n",
       "  'sigmaB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'ydaP',\n",
       "  'bmrUR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T5',\n",
       "  'T8',\n",
       "  'sigmaB',\n",
       "  'yfkM',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T5',\n",
       "  'T1',\n",
       "  'sigmaB',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T5',\n",
       "  'T3',\n",
       "  'sigmaB',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T5',\n",
       "  'T2',\n",
       "  'sigmaB',\n",
       "  'bmrUR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T8',\n",
       "  'T1',\n",
       "  'yfkM',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T8',\n",
       "  'T3',\n",
       "  'yfkM',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T8',\n",
       "  'T2',\n",
       "  'yfkM',\n",
       "  'bmrUR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'sigmaB',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigmaB',\n",
       "  'bmrUR',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d16',\n",
       "  '(i) The finding that the bmrUR operon is under sigmaB control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\\nThe ydaP gene might belong to the increasing number of sigmaB-dependent genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\\nThree new sigmaB-dependent genes (ydaE, ydaG and yfkM) encoding proteins with still unknown functions were also described.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'sigmaB',\n",
       "  'bmrUR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d33',\n",
       "  'During endospore formation in Bacillus subtilis, the DNA binding protein GerE stimulates transcription from several promoters that are used by RNA polymerase containing sigmaK\\nGerE binds to a site on one of these promoters, cotX, that overlaps its -35 region.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'cotX',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d33',\n",
       "  'During endospore formation in Bacillus subtilis, the DNA binding protein GerE stimulates transcription from several promoters that are used by RNA polymerase containing sigmaK\\nGerE binds to a site on one of these promoters, cotX, that overlaps its -35 region.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'cotX',\n",
       "  'GerE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d33',\n",
       "  'During endospore formation in Bacillus subtilis, the DNA binding protein GerE stimulates transcription from several promoters that are used by RNA polymerase containing sigmaK\\nGerE binds to a site on one of these promoters, cotX, that overlaps its -35 region.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'cotX',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d33',\n",
       "  'During endospore formation in Bacillus subtilis, the DNA binding protein GerE stimulates transcription from several promoters that are used by RNA polymerase containing sigmaK\\nGerE binds to a site on one of these promoters, cotX, that overlaps its -35 region.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'GerE',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d33',\n",
       "  'During endospore formation in Bacillus subtilis, the DNA binding protein GerE stimulates transcription from several promoters that are used by RNA polymerase containing sigmaK\\nGerE binds to a site on one of these promoters, cotX, that overlaps its -35 region.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'GerE',\n",
       "  'sigmaK',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d33',\n",
       "  'During endospore formation in Bacillus subtilis, the DNA binding protein GerE stimulates transcription from several promoters that are used by RNA polymerase containing sigmaK\\nGerE binds to a site on one of these promoters, cotX, that overlaps its -35 region.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'GerE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d34',\n",
       "  'However, the mutant was unable to stimulate transcription by final sigma(A)-RNA polymerase from the Spo0A-dependent spoIIG operon promoter.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'Spo0A',\n",
       "  'spoIIG',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d34',\n",
       "  'However, the mutant was unable to stimulate transcription by final sigma(A)-RNA polymerase from the Spo0A-dependent spoIIG operon promoter.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'Spo0A',\n",
       "  'sigma(A)',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d34',\n",
       "  'However, the mutant was unable to stimulate transcription by final sigma(A)-RNA polymerase from the Spo0A-dependent spoIIG operon promoter.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'spoIIG',\n",
       "  'sigma(A)',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T6',\n",
       "  'T7',\n",
       "  'sigma G',\n",
       "  'sigma 28',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T6',\n",
       "  'T4',\n",
       "  'sigma G',\n",
       "  'DnaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T6',\n",
       "  'T5',\n",
       "  'sigma G',\n",
       "  'sigma F',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T6',\n",
       "  'T8',\n",
       "  'sigma G',\n",
       "  'sigma B',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T6',\n",
       "  'T9',\n",
       "  'sigma G',\n",
       "  'sigma 32',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T6',\n",
       "  'T1',\n",
       "  'sigma G',\n",
       "  'SpoIIAB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T6',\n",
       "  'T3',\n",
       "  'sigma G',\n",
       "  'RbsW',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T6',\n",
       "  'T2',\n",
       "  'sigma G',\n",
       "  'FlgM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T7',\n",
       "  'T4',\n",
       "  'sigma 28',\n",
       "  'DnaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T7',\n",
       "  'T5',\n",
       "  'sigma 28',\n",
       "  'sigma F',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T7',\n",
       "  'T8',\n",
       "  'sigma 28',\n",
       "  'sigma B',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T7',\n",
       "  'T9',\n",
       "  'sigma 28',\n",
       "  'sigma 32',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T7',\n",
       "  'T1',\n",
       "  'sigma 28',\n",
       "  'SpoIIAB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T7',\n",
       "  'T3',\n",
       "  'sigma 28',\n",
       "  'RbsW',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T7',\n",
       "  'T2',\n",
       "  'sigma 28',\n",
       "  'FlgM',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T4',\n",
       "  'T5',\n",
       "  'DnaK',\n",
       "  'sigma F',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T4',\n",
       "  'T8',\n",
       "  'DnaK',\n",
       "  'sigma B',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T4',\n",
       "  'T9',\n",
       "  'DnaK',\n",
       "  'sigma 32',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'DnaK',\n",
       "  'SpoIIAB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'DnaK',\n",
       "  'RbsW',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'DnaK',\n",
       "  'FlgM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T5',\n",
       "  'T8',\n",
       "  'sigma F',\n",
       "  'sigma B',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T5',\n",
       "  'T9',\n",
       "  'sigma F',\n",
       "  'sigma 32',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T5',\n",
       "  'T1',\n",
       "  'sigma F',\n",
       "  'SpoIIAB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T5',\n",
       "  'T3',\n",
       "  'sigma F',\n",
       "  'RbsW',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T5',\n",
       "  'T2',\n",
       "  'sigma F',\n",
       "  'FlgM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T8',\n",
       "  'T9',\n",
       "  'sigma B',\n",
       "  'sigma 32',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T8',\n",
       "  'T1',\n",
       "  'sigma B',\n",
       "  'SpoIIAB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T8',\n",
       "  'T3',\n",
       "  'sigma B',\n",
       "  'RbsW',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T8',\n",
       "  'T2',\n",
       "  'sigma B',\n",
       "  'FlgM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T9',\n",
       "  'T1',\n",
       "  'sigma 32',\n",
       "  'SpoIIAB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T9',\n",
       "  'T3',\n",
       "  'sigma 32',\n",
       "  'RbsW',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T9',\n",
       "  'T2',\n",
       "  'sigma 32',\n",
       "  'FlgM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'SpoIIAB',\n",
       "  'RbsW',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'SpoIIAB',\n",
       "  'FlgM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d41',\n",
       "  'The class includes AsiA form bacteriophage T4, which inhibits Escherichia coli sigma 70; FlgM, present in both gram-positive and gram-negative bacteria, which inhibits the flagella sigma factor sigma 28; SpoIIAB, which inhibits the sporulation-specific sigma factor, sigma F and sigma G, of Bacillus subtilis; RbsW of B. subtilis, which inhibits stress response sigma factor sigma B; and DnaK, a general regulator of the heat shock response, which in bacteria inhibits the heat shock sigma factor sigma 32.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'RbsW',\n",
       "  'FlgM',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T6',\n",
       "  'T7',\n",
       "  'sigmaF',\n",
       "  'katX',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T6',\n",
       "  'T4',\n",
       "  'sigmaF',\n",
       "  'katX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T6',\n",
       "  'T5',\n",
       "  'sigmaF',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T6',\n",
       "  'T8',\n",
       "  'sigmaF',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T6',\n",
       "  'T9',\n",
       "  'sigmaF',\n",
       "  'katX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T6',\n",
       "  'T1',\n",
       "  'sigmaF',\n",
       "  'EsigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T6',\n",
       "  'T3',\n",
       "  'sigmaF',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T6',\n",
       "  'T2',\n",
       "  'sigmaF',\n",
       "  'KatX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T7',\n",
       "  'T4',\n",
       "  'katX',\n",
       "  'katX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T7',\n",
       "  'T5',\n",
       "  'katX',\n",
       "  'sigmaB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T7',\n",
       "  'T8',\n",
       "  'katX',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T7',\n",
       "  'T9',\n",
       "  'katX',\n",
       "  'katX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T7',\n",
       "  'T1',\n",
       "  'katX',\n",
       "  'EsigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T7',\n",
       "  'T3',\n",
       "  'katX',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T7',\n",
       "  'T2',\n",
       "  'katX',\n",
       "  'KatX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T4',\n",
       "  'T5',\n",
       "  'katX',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T4',\n",
       "  'T8',\n",
       "  'katX',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T4',\n",
       "  'T9',\n",
       "  'katX',\n",
       "  'katX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'katX',\n",
       "  'EsigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'katX',\n",
       "  'sigmaB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'katX',\n",
       "  'KatX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T5',\n",
       "  'T8',\n",
       "  'sigmaB',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T5',\n",
       "  'T9',\n",
       "  'sigmaB',\n",
       "  'katX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T5',\n",
       "  'T1',\n",
       "  'sigmaB',\n",
       "  'EsigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T5',\n",
       "  'T3',\n",
       "  'sigmaB',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T5',\n",
       "  'T2',\n",
       "  'sigmaB',\n",
       "  'KatX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T8',\n",
       "  'T9',\n",
       "  'sigmaB',\n",
       "  'katX',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T8',\n",
       "  'T1',\n",
       "  'sigmaB',\n",
       "  'EsigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T8',\n",
       "  'T3',\n",
       "  'sigmaB',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T8',\n",
       "  'T2',\n",
       "  'sigmaB',\n",
       "  'KatX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T9',\n",
       "  'T1',\n",
       "  'katX',\n",
       "  'EsigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T9',\n",
       "  'T3',\n",
       "  'katX',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T9',\n",
       "  'T2',\n",
       "  'katX',\n",
       "  'KatX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'EsigmaF',\n",
       "  'sigmaB',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'EsigmaF',\n",
       "  'KatX',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d8',\n",
       "  'Our results indicate that the level of KatX level in outgrowing spores depends mainly on EsigmaF, because sigB mutants show normal KatX activity in dormant and outgrowing spores.\\nTherefore, the physiological role of sigmaB-dependent katX expression remains obscure.\\nThe -10 and -35 sequences of the sigmaB- and sigmaF-dependent promoters of katX overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\\nWe demonstrated that katX is also a sigmaB-dependent general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'sigmaB',\n",
       "  'KatX',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d1',\n",
       "  'In this mutant, expression of the spoIIG gene, whose transcription depends on both sigma(A) and the phosphorylated Spo0A protein, Spo0A~P, a major transcription factor during early stages of sporulation, was greatly reduced at 43 degrees C.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'sigma(A)',\n",
       "  'spoIIG',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d1',\n",
       "  'In this mutant, expression of the spoIIG gene, whose transcription depends on both sigma(A) and the phosphorylated Spo0A protein, Spo0A~P, a major transcription factor during early stages of sporulation, was greatly reduced at 43 degrees C.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigma(A)',\n",
       "  'Spo0A',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d1',\n",
       "  'In this mutant, expression of the spoIIG gene, whose transcription depends on both sigma(A) and the phosphorylated Spo0A protein, Spo0A~P, a major transcription factor during early stages of sporulation, was greatly reduced at 43 degrees C.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'spoIIG',\n",
       "  'Spo0A',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d6',\n",
       "  'The expression of rsfA is under the control of both sigma(F) and sigma(G).',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'sigma(F)',\n",
       "  'rsfA',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d6',\n",
       "  'The expression of rsfA is under the control of both sigma(F) and sigma(G).',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigma(F)',\n",
       "  'sigma(G)',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d6',\n",
       "  'The expression of rsfA is under the control of both sigma(F) and sigma(G).',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'rsfA',\n",
       "  'sigma(G)',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d17',\n",
       "  'Cell specificity is first established in the forespore under the control of the sigma F factor, which is itself activated through the action of the SpoIIE serine phosphatase, an enzyme targeted to the septum between the two cells.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'SpoIIE',\n",
       "  'sigma F',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T4',\n",
       "  'T5',\n",
       "  'sigma(54)',\n",
       "  'rocG',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'sigma(54)',\n",
       "  'SigL',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'sigma(54)',\n",
       "  'rocG',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'sigma(54)',\n",
       "  'RocR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T5',\n",
       "  'T1',\n",
       "  'rocG',\n",
       "  'SigL',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T5',\n",
       "  'T3',\n",
       "  'rocG',\n",
       "  'rocG',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T5',\n",
       "  'T2',\n",
       "  'rocG',\n",
       "  'RocR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'SigL',\n",
       "  'rocG',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'SigL',\n",
       "  'RocR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d10',\n",
       "  'The rocG gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by SigL-containing RNA polymerase and requires for its expression RocR, a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\\nUnlike the case for other sigma(54)-dependent genes, rocG has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'rocG',\n",
       "  'RocR',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d19',\n",
       "  'sspG transcription also requires the DNA binding protein GerE.\\nIn contrast, sspJ is transcribed in the forespore compartment by RNA polymerase with the forespore-specific sigmaG and appears to give a monocistronic transcript.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'sspJ',\n",
       "  'GerE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d19',\n",
       "  'sspG transcription also requires the DNA binding protein GerE.\\nIn contrast, sspJ is transcribed in the forespore compartment by RNA polymerase with the forespore-specific sigmaG and appears to give a monocistronic transcript.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'sspJ',\n",
       "  'sigmaG',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d19',\n",
       "  'sspG transcription also requires the DNA binding protein GerE.\\nIn contrast, sspJ is transcribed in the forespore compartment by RNA polymerase with the forespore-specific sigmaG and appears to give a monocistronic transcript.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'sspJ',\n",
       "  'sspG',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d19',\n",
       "  'sspG transcription also requires the DNA binding protein GerE.\\nIn contrast, sspJ is transcribed in the forespore compartment by RNA polymerase with the forespore-specific sigmaG and appears to give a monocistronic transcript.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'GerE',\n",
       "  'sigmaG',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d19',\n",
       "  'sspG transcription also requires the DNA binding protein GerE.\\nIn contrast, sspJ is transcribed in the forespore compartment by RNA polymerase with the forespore-specific sigmaG and appears to give a monocistronic transcript.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'GerE',\n",
       "  'sspG',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d19',\n",
       "  'sspG transcription also requires the DNA binding protein GerE.\\nIn contrast, sspJ is transcribed in the forespore compartment by RNA polymerase with the forespore-specific sigmaG and appears to give a monocistronic transcript.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'sigmaG',\n",
       "  'sspG',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d25',\n",
       "  'The expression of the bkd operon was induced by the presence of isoleucine or valine in the growth medium and depended upon the presence of the sigma factor SigL, a member of the sigma 54 family.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'SigL',\n",
       "  'bkd',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d22',\n",
       "  'sigma X also contributes to the expression of CsbB, a putative membrane-bound glucosyl transferase that is partially controlled by the sigma B stress response sigma factor.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'sigma X',\n",
       "  'CsbB',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d22',\n",
       "  'sigma X also contributes to the expression of CsbB, a putative membrane-bound glucosyl transferase that is partially controlled by the sigma B stress response sigma factor.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigma X',\n",
       "  'sigma B',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d22',\n",
       "  'sigma X also contributes to the expression of CsbB, a putative membrane-bound glucosyl transferase that is partially controlled by the sigma B stress response sigma factor.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'CsbB',\n",
       "  'sigma B',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T6',\n",
       "  'T7',\n",
       "  'ald',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T6',\n",
       "  'T4',\n",
       "  'ald',\n",
       "  'sigmaA',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T6',\n",
       "  'T10',\n",
       "  'ald',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T6',\n",
       "  'T5',\n",
       "  'ald',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T6',\n",
       "  'T8',\n",
       "  'ald',\n",
       "  'sigmaA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T6',\n",
       "  'T9',\n",
       "  'ald',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T6',\n",
       "  'T1',\n",
       "  'ald',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T6',\n",
       "  'T3',\n",
       "  'ald',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T6',\n",
       "  'T2',\n",
       "  'ald',\n",
       "  'Spo0A',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T7',\n",
       "  'T4',\n",
       "  'sigE',\n",
       "  'sigmaA',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T7',\n",
       "  'T10',\n",
       "  'sigE',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T7',\n",
       "  'T5',\n",
       "  'sigE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T7',\n",
       "  'T8',\n",
       "  'sigE',\n",
       "  'sigmaA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T7',\n",
       "  'T9',\n",
       "  'sigE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T7',\n",
       "  'T1',\n",
       "  'sigE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T7',\n",
       "  'T3',\n",
       "  'sigE',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T7',\n",
       "  'T2',\n",
       "  'sigE',\n",
       "  'Spo0A',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T4',\n",
       "  'T10',\n",
       "  'sigmaA',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T4',\n",
       "  'T5',\n",
       "  'sigmaA',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T4',\n",
       "  'T8',\n",
       "  'sigmaA',\n",
       "  'sigmaA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T4',\n",
       "  'T9',\n",
       "  'sigmaA',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'sigmaA',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'sigmaA',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'sigmaA',\n",
       "  'Spo0A',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T10',\n",
       "  'T5',\n",
       "  'sigE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T10',\n",
       "  'T8',\n",
       "  'sigE',\n",
       "  'sigmaA',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T10',\n",
       "  'T9',\n",
       "  'sigE',\n",
       "  'sigmaK',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T10',\n",
       "  'T1',\n",
       "  'sigE',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T10',\n",
       "  'T3',\n",
       "  'sigE',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T10',\n",
       "  'T2',\n",
       "  'sigE',\n",
       "  'Spo0A',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T5',\n",
       "  'T8',\n",
       "  'sigmaK',\n",
       "  'sigmaA',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T5',\n",
       "  'T9',\n",
       "  'sigmaK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T5',\n",
       "  'T1',\n",
       "  'sigmaK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T5',\n",
       "  'T3',\n",
       "  'sigmaK',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T5',\n",
       "  'T2',\n",
       "  'sigmaK',\n",
       "  'Spo0A',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T8',\n",
       "  'T9',\n",
       "  'sigmaA',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T8',\n",
       "  'T1',\n",
       "  'sigmaA',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T8',\n",
       "  'T3',\n",
       "  'sigmaA',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T8',\n",
       "  'T2',\n",
       "  'sigmaA',\n",
       "  'Spo0A',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T9',\n",
       "  'T1',\n",
       "  'sigmaK',\n",
       "  'sigmaK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T9',\n",
       "  'T3',\n",
       "  'sigmaK',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T9',\n",
       "  'T2',\n",
       "  'sigmaK',\n",
       "  'Spo0A',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'sigmaK',\n",
       "  'sigE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigmaK',\n",
       "  'Spo0A',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d13',\n",
       "  'Production of sigmaK about 1 h earlier than normal does affect Spo0A, which when phosphorylated is an activator of sigE transcription.\\nThe ald promoter, like the sigE promoter, is believed to be recognized by sigmaA RNA polymerase, suggesting that sigmaK may inhibit sigmaA activity late in sporulation.\\nIn the mother cell compartment of sporulating cells, expression of the sigE gene, encoding the earlier-acting sigma factor, sigmaE, is negatively regulated by the later-acting sigma factor, sigmaK.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'sigE',\n",
       "  'Spo0A',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T7',\n",
       "  'SpoIIAA',\n",
       "  'sigmaF',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T4',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T10',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T5',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T11',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIAA-P',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T12',\n",
       "  'SpoIIAA',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T8',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T9',\n",
       "  'SpoIIAA',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T1',\n",
       "  'SpoIIAA',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T3',\n",
       "  'SpoIIAA',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T6',\n",
       "  'T2',\n",
       "  'SpoIIAA',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T4',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T10',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T5',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T11',\n",
       "  'sigmaF',\n",
       "  'SpoIIAA-P',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T12',\n",
       "  'sigmaF',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T8',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T9',\n",
       "  'sigmaF',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T1',\n",
       "  'sigmaF',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T3',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T7',\n",
       "  'T2',\n",
       "  'sigmaF',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T4',\n",
       "  'T10',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T4',\n",
       "  'T5',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T4',\n",
       "  'T11',\n",
       "  'SpoIIE',\n",
       "  'SpoIIAA-P',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T4',\n",
       "  'T12',\n",
       "  'SpoIIE',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T4',\n",
       "  'T8',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T4',\n",
       "  'T9',\n",
       "  'SpoIIE',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T4',\n",
       "  'T1',\n",
       "  'SpoIIE',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T4',\n",
       "  'T3',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T4',\n",
       "  'T2',\n",
       "  'SpoIIE',\n",
       "  'FtsZ',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T10',\n",
       "  'T5',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T10',\n",
       "  'T11',\n",
       "  'SpoIIE',\n",
       "  'SpoIIAA-P',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T10',\n",
       "  'T12',\n",
       "  'SpoIIE',\n",
       "  'sigmaF',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T10',\n",
       "  'T8',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T10',\n",
       "  'T9',\n",
       "  'SpoIIE',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T10',\n",
       "  'T1',\n",
       "  'SpoIIE',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T10',\n",
       "  'T3',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T10',\n",
       "  'T2',\n",
       "  'SpoIIE',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T5',\n",
       "  'T11',\n",
       "  'SpoIIE',\n",
       "  'SpoIIAA-P',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T5',\n",
       "  'T12',\n",
       "  'SpoIIE',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T5',\n",
       "  'T8',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T5',\n",
       "  'T9',\n",
       "  'SpoIIE',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T5',\n",
       "  'T1',\n",
       "  'SpoIIE',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T5',\n",
       "  'T3',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T5',\n",
       "  'T2',\n",
       "  'SpoIIE',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T11',\n",
       "  'T12',\n",
       "  'SpoIIAA-P',\n",
       "  'sigmaF',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T11',\n",
       "  'T8',\n",
       "  'SpoIIAA-P',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T11',\n",
       "  'T9',\n",
       "  'SpoIIAA-P',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T11',\n",
       "  'T1',\n",
       "  'SpoIIAA-P',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T11',\n",
       "  'T3',\n",
       "  'SpoIIAA-P',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T11',\n",
       "  'T2',\n",
       "  'SpoIIAA-P',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T12',\n",
       "  'T8',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T12',\n",
       "  'T9',\n",
       "  'sigmaF',\n",
       "  'sigmaF',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T12',\n",
       "  'T1',\n",
       "  'sigmaF',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T12',\n",
       "  'T3',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T12',\n",
       "  'T2',\n",
       "  'sigmaF',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T8',\n",
       "  'T9',\n",
       "  'SpoIIE',\n",
       "  'sigmaF',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T8',\n",
       "  'T1',\n",
       "  'SpoIIE',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T8',\n",
       "  'T3',\n",
       "  'SpoIIE',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T8',\n",
       "  'T2',\n",
       "  'SpoIIE',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T9',\n",
       "  'T1',\n",
       "  'sigmaF',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T9',\n",
       "  'T3',\n",
       "  'sigmaF',\n",
       "  'SpoIIE',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T9',\n",
       "  'T2',\n",
       "  'sigmaF',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T1',\n",
       "  'T3',\n",
       "  'FtsZ',\n",
       "  'SpoIIE',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'FtsZ',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d14',\n",
       "  'The first stage, which involves the phosphatase function of SpoIIE, depends on the cell division protein FtsZ and could correspond to the FtsZ-dependent assembly of SpoIIE into E-rings.\\nEvidence based on the use of modified and mutant forms of the phosphatase protein indicates that SpoIIE blocks the capacity of unphosphorylated SpoIIAA to activate sigmaF until formation of the polar septum is completed.\\nCell-specific activation of transcription factor sigmaF during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase (SpoIIE) located in the septum.\\nThe SpoIIE phosphatase indirectly activates sigmaF by dephosphorylating a protein (SpoIIAA-P) in the pathway that controls the activity of the transcription factor.',\n",
       "  'T3',\n",
       "  'T2',\n",
       "  'SpoIIE',\n",
       "  'FtsZ',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d28',\n",
       "  'Primer extension experiments and Northern blot analysis show that an active sigmaA-dependent promoter precedes kdgR and transcription is terminated at the putative p-independent terminator downstream of kdgT.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigmaA',\n",
       "  'kdgR',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d21',\n",
       "  'This stress-induced increase in ykzA expression is absolutely dependent on the alternative sigma factor sigmaB, which controls a large stationary-phase and stress regulon.',\n",
       "  'T1',\n",
       "  'T2',\n",
       "  'sigmaB',\n",
       "  'ykzA',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d26',\n",
       "  'From these results we conclude that ComK negatively regulates degR expression by preventing sigmaD-driven transcription of degR, possibly through interaction with the control region.\\nWe have shown previously that the transcription of degR is driven by an alternative sigma factor, sigmaD.',\n",
       "  'T6',\n",
       "  'T4',\n",
       "  'degR',\n",
       "  'degR',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ['LLL_d26',\n",
       "  'From these results we conclude that ComK negatively regulates degR expression by preventing sigmaD-driven transcription of degR, possibly through interaction with the control region.\\nWe have shown previously that the transcription of degR is driven by an alternative sigma factor, sigmaD.',\n",
       "  'T6',\n",
       "  'T5',\n",
       "  'degR',\n",
       "  'sigmaD',\n",
       "  'TRUE'],\n",
       " [],\n",
       " ['LLL_d26',\n",
       "  'From these results we conclude that ComK negatively regulates degR expression by preventing sigmaD-driven transcription of degR, possibly through interaction with the control region.\\nWe have shown previously that the transcription of degR is driven by an alternative sigma factor, sigmaD.',\n",
       "  'T6',\n",
       "  'T1',\n",
       "  'degR',\n",
       "  'ComK',\n",
       "  'FALSE'],\n",
       " [],\n",
       " ...]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "colab": {
   "authorship_tag": "ABX9TyMJi7e4Uz1bLMDtV4lPFUbM",
   "name": "Medical_NER.ipynb",
   "provenance": []
  },
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
